src/pages/Login/Login.module.css

```diff
@@ -34,3 +34,40 @@
     justify-content: center;
 }
 
+
+.tooltipContainer {
+    position: relative;
+    display: inline-block;
+}
+
+.tooltipContainer .tooltipText {
+    visibility: hidden;
+    width: 120px;
+    background-color: gray;
+    color: #fff;
+    text-align: center;
+    padding: 5px 0;
+    border-radius: 6px;
+    position: absolute;
+    z-index: 1;
+    width: 150px;
+    bottom: 100%;
+    left: 50%;
+    margin-left: -60px;
+}
+
+.tooltipContainer .tooltipText:after {
+    content: " ";
+    position: absolute;
+    top: 100%;
+    left: 50%;
+    margin-left: -5px;
+    border-width: 5px;
```

Comment:
I'd recommend to make this dependency of border-width and margin-left more clear by using the same variable. You can even remove negative margin and deduct 5px from `left: 50%`. CSS custom properties will help:
```css
    --border-width: 5px;
    left: calc(50% - var(--border-width));
    border-width: var(--border-width);
```

Will this do the trick?